apps/google-analytics-4/frontend/src/components/main-app/AnalyticsApp/AnalyticsApp.tsx

```diff
@@ -53,9 +56,9 @@ const AnalyticsApp = (props: Props) => {
       setLoading(false);
     }
 
-    if (reportSlug && propertyId && !isContentTypeWarning) fetchRunReportData();
-    else setTimeout(() => setLoading(false), 200);
-  }, [api, reportRequestParams, reportSlug, propertyId, isContentTypeWarning]);
+    if (runReportFetchRequirements) fetchRunReportData();
+    else setLoading(false);
```

Comment:
nit: You can just remove line 56 and remove this else and just have it `setLoading(false)` everytime

Reply:
nice, great catch, have adjusted 💯